src/BUILD_WINDOWS.md

```diff
@@ -40,7 +40,43 @@ If using anything else other than Visual Studio 2008 for development, your code
 
 It is OK to add newer Visual Studio (2015, 2017) solution files to the project, but there then must be dual solution files for both Visual C++ 2008 and the latest Visual Studio.
 
-Note: There is an update to the CMake configuration that adds support for Windows in the works for future use.
+Build and Development Instructions with Visual Studio 2017 & CMake
+---------------------------------------------------------------------
+
+An alternative method for development of the SoftEtherVPN project on Windows is through CMake.
+
+There are several methods for using CMake but the easiest by far is through Visual Studio 2017 by importing the CMake project directly
+into it. So that is what will be described below.
+
+Requirements:
+
+1. Download Visual Studio 2017 (Community Edition is fine).
+2. During install, make sure to check "Desktop development with C++" under "Workloads".
+3. Click on individual components and scroll until you see "Visual C++ tools for CMake" under the compilers section. Make sure this is checked.
+4. Continue scrolling down until you reach "SDKs, libraries, and frameworks". Under this tab you want to check to options:
```

Comment:
can we add command line installation instrruction ?

something based on workload is

https://docs.microsoft.com/en-us/visualstudio/install/workload-and-component-ids?view=vs-2017

Reply:
I mean, when VS2017 is installed, you can add workloads to existing installation

Reply:
I definitely could add something about just adding it as another workload the process is dead identical as VS2017 installs and modifies through the Visual Studio Installer application.

I would have to research the CLI instructions, as I have never touched them before. Do standard Windows developers use them that frequently?

Reply:
we use things like that in automated build agent setup. if you think developers do not use it frequently, let's ignore that

Reply:
At the moment, the AppVeyor has a Visual Studio 2017 build process, so I assume its already setup in some way?